### PR TITLE
create layout #17

### PR DESCRIPTION
**Pull request by ZinabHashem**

### CODE REVIEW

File: ui/src/app/core/core.module.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    FooterComponent,
+    MainComponent,
+    SideBarListComponent,
+    ],
+  imports: [
+    CommonModule,
+    SharedModule,
+    RouterModule
+ ],
+  providers: [],
```
Comment (diaahamad) [inline ui/src/app/core/core.module.ts:22]: remove semicolon

File: ui/src/index.html [hunk 2]
```diff
@@ -6,6 +6,7 @@
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```
Comment (diaahamad) [inline ui/src/index.html:9]: this will not work if the internet down

Comment (ZinabHashem) [reply ui/src/index.html]: modified

File: ui/src/app/core/core.module.ts (hunk 1 above)
Comment (ZinabHashem) [reply ui/src/app/core/core.module.ts]: done

**Pull request merged**